### PR TITLE
Removed nonce from KeyInfo

### LINKED ISSUE #193: [Storage Optimization] Remove KeyInfo Struct
Due to the change in how we use MSA keys, we no longer need to store any additional data along with the MSA Id when mapping from Account Id to MSA Id (aka what is currently Account Id -> KeyInfo )

- [ ] Replace KeyInfo with just the MSA Id
- [ ] Update RPCs that returned that information
- [ ] Update js/api-augment
- [ ] Update weights

### PR DESCRIPTION
**Pull request by dopatraman**
# Goal
The goal of this PR is to remove the `nonce` field from the `KeyInfo` class.

Closes #193 

# Discussion
We no longer need the `nonce` field to prevent Replay Attacks on adding keys.

# Checklist
- [ ] Updated the js/api-augment code if a custom RPC added/changed
- [x] Tests updated
- [ ] Weights updated

### CODE REVIEW

File: pallets/msa/src/weights.rs
Comment (shannonwells) [inline pallets/msa/src/weights.rs:91]: That's weird, I'd think removing data from this struct would lower the weight. I guess it's your hardware.

**Review by shannonwells — APPROVED** [review]
Seems legit

**Comment by enddynayn** [comment]
I do not think we want to remove nonce right? Can we hold off till tomorrow before merging.

**Review by saraswatpuneet — CHANGES_REQUESTED** [review]
Keeping it as blocker. I do not think we confirmed removing nonce yet

**Comment by wilwade** [comment]
One other note. If we do remove nonce and expired (#308), then we likely want to remove KeyInfo struct entirely.

**Comment by wilwade** [comment]
Closing as https://github.com/LibertyDSNP/frequency/issues/193 has switched up.

**Pull request closed without merging**
